resources/quality/eryone_thinker/eryone_thinker_fast.inst.cfg

```diff
@@ -1,33 +0,0 @@
-[general]
```

Comment:
Deleting a file requires an upgrade script; otherwise the configuration would become invalid.

Reply:
Hi, this file was removed by mistake. (It has had a conflict problem before so we removed it then it works. Now we updated it)
Thanks!

Reply:
Now that the delete has be resolve, anything blocking this merge?